quickwit/quickwit-control-plane/src/ingest/ingest_controller.rs

```diff
@@ -706,9 +780,189 @@ impl IngestController {
         }
     }
 
-    pub fn rebalance_shards(&mut self) {
-        // TODO: As of now, it is only used for unit testing.
+    pub(crate) async fn rebalance_shards(
```

Comment:
comment on this method please.

This is not that trivial as it seem for an outsider I think
rebalance shards could mean something about indexer rebalancing.